apps/condo/migrations/20210629152912-0024_auto_20210629_1029.js

```diff
@@ -0,0 +1,49 @@
+// auto generated by kmigrator
+// KMIGRATOR:0024_auto_20210629_1029:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMiBvbiAyMDIxLTA2LTI5IDEwOjI5Cgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucywgbW9kZWxzCmltcG9ydCBkamFuZ28uZGIubW9kZWxzLmRlbGV0aW9uCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMDIzX21lcmdlXzIwMjEwNjI5XzEwMjcnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWx0ZXJGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nY29udGFjdCcsCiAgICAgICAgICAgIG5hbWU9J3Byb3BlcnR5JywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkZvcmVpZ25LZXkoZGJfY29sdW1uPSdwcm9wZXJ0eScsIGRlZmF1bHQ9J2Q4ODY1MWQ1LWIzMTEtNGVjZi04MmFjLWI1Yjg3ZmVjZDA2ZicsIG9uX2RlbGV0ZT1kamFuZ28uZGIubW9kZWxzLmRlbGV0aW9uLkNBU0NBREUsIHJlbGF0ZWRfbmFtZT0nKycsIHRvPSdfZGphbmdvX3NjaGVtYS5wcm9wZXJ0eScpLAogICAgICAgICAgICBwcmVzZXJ2ZV9kZWZhdWx0PUZhbHNlLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BbHRlckZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdmb3Jnb3RwYXNzd29yZGFjdGlvbicsCiAgICAgICAgICAgIG5hbWU9J3Rva2VuJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZCgpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Alter field property on contact
+--
+
+-- HOTFIX: MISSED migration hack!
+delete from "Contact" where property is NULL;
+COMMIT;
+
+BEGIN;
+SET CONSTRAINTS "Contact_property_517359e1_fk_Property_id" IMMEDIATE; ALTER TABLE "Contact" DROP CONSTRAINT "Contact_property_517359e1_fk_Property_id";
+ALTER TABLE "Contact" ALTER COLUMN "property" SET DEFAULT 'd88651d5-b311-4ecf-82ac-b5b87fecd06f'::uuid;
+UPDATE "Contact" SET "property" = 'd88651d5-b311-4ecf-82ac-b5b87fecd06f'::uuid WHERE "property" IS NULL;
+ALTER TABLE "Contact" ALTER COLUMN "property" SET NOT NULL;
+ALTER TABLE "Contact" ADD CONSTRAINT "Contact_property_517359e1_fk_Property_id" FOREIGN KEY ("property") REFERENCES "Property" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "Contact" ALTER COLUMN "property" DROP DEFAULT;
```

Comment:
It drops default just after it's being set in before coming line.

Reply:
it's autogenerated SQL. it's ok